app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsController < ApplicationController
+  def index
+    food_input          = params[:q]
+    if food_input       == ""
+     flash[:alert]      = "Please input a food item to search!"
+     redirect_to root_path
+    else
+      @foods_info       = FoodsFacade.search_by_food(food_input)
+      @total_food_count = FoodsFacade.searched_food_count(food_input)
```

Comment:
It's totally fine to pass two instance variables to the view in this case, but it would be best to only make one request to the API, because we can get all the info we need with one request.